frontend/rollup.config.js

```diff
@@ -1,75 +1,71 @@
-import svelte from 'rollup-plugin-svelte';
-import resolve from '@rollup/plugin-node-resolve';
-import commonjs from '@rollup/plugin-commonjs';
-import livereload from 'rollup-plugin-livereload';
-import { terser } from 'rollup-plugin-terser';
+import fs from "fs-extra"
+import commonjs from "@rollup/plugin-commonjs"
+import resolve from "@rollup/plugin-node-resolve"
+import livereload from "rollup-plugin-livereload"
+import svelte from "rollup-plugin-svelte"
+import { terser } from "rollup-plugin-terser"
 
-const production = !process.env.ROLLUP_WATCH;
-
-function serve() {
-	let server;
-	
-	function toExit() {
-		if (server) server.kill(0);
-	}
-
-	return {
-		writeBundle() {
-			if (server) return;
-			server = require('child_process').spawn('npm', ['run', 'start', '--', '--dev'], {
-				stdio: ['ignore', 'inherit', 'inherit'],
-				shell: true
-			});
-
-			process.on('SIGTERM', toExit);
-			process.on('exit', toExit);
-		}
-	};
-}
+const production = !process.env.ROLLUP_WATCH
 
 export default {
-	input: 'src/main.js',
+	input: "src/main.js",
 	output: {
 		sourcemap: true,
-		format: 'iife',
-		name: 'app',
-		file: 'public/build/bundle.js'
+		format: "iife",
+		name: "app",
+		file: "public/build/bundle.js"
 	},
 	plugins: [
 		svelte({
-			// enable run-time checks when not in production
 			dev: !production,
-			// we'll extract any component CSS out into
-			// a separate file - better for performance
-			css: css => {
-				css.write('public/build/bundle.css');
-			}
+			css: css => css.write("public/build/bundle.css")
```

Comment:
this looks like a slightly odd change, but i don't understand it really. it was returning an object with the value of css.write() in it (i dont know what the key is), and now returns the result of css.write(). but ok. just commenting how I dont understand.